### PR TITLE
Add HITCON16 Sleepy Holder and Heart Attack writeups

### PR DESCRIPTION
**Pull request by andigena**

### CODE REVIEW

File: hitcon-ctf-2016/pwn/heart-attack-400/README.md [hunk 1]
```diff
@@ -14,4 +14,4 @@
 
 ## Other write-ups and resources
 
-* none yet
+* [Writeup by the challenge author](https://github.com/mehQQ/public_writeup/tree/master/hitcon2016/HeartAttack)
```
Comment (tjgerot) [inline hitcon-ctf-2016/pwn/heart-attack-400/README.md]: May I suggest
`[NAME (Challenge Author)](https://github.com/mehQQ/public_writeup/tree/master/hitcon2016/HeartAttack)`
where you replace `NAME` with your own?

File: hitcon-ctf-2016/pwn/sleepy-holder-300/README.md [hunk 2]
```diff
@@ -14,4 +14,4 @@
 
 ## Other write-ups and resources
 
-* none yet
+* [Writeup by the challenge author](https://github.com/mehQQ/public_writeup/tree/master/hitcon2016/SleepyHolder)
```
Comment (tjgerot) [inline hitcon-ctf-2016/pwn/sleepy-holder-300/README.md]: Same suggestion as above.

**Review by ghost — CHANGES_REQUESTED** [review]
Except for small fixes in the link formatting, looks pretty good!

File: hitcon-ctf-2016/pwn/heart-attack-400/README.md (hunk 1 above)
Comment (andigena) [inline hitcon-ctf-2016/pwn/heart-attack-400/README.md]: Sure, I've made the changes.

**Review by ghost — APPROVED** [review]

**Comment by tjgerot** [comment]
Thanks! It's always great to get write-ups from the author. Happy Thanksgiving.

**Pull request merged**